src/qt/test/rpcnestedtests.cpp

```diff
@@ -154,10 +154,14 @@ void RPCNestedTests::rpcNestedTests()
     QVERIFY_EXCEPTION_THROWN(RPCConsole::RPCExecuteCommandLine(result, "rpcNestedTest(abc,,)"), std::runtime_error); //don't tollerate empty arguments when using ,
 #endif
 
+    UnloadBlockIndex();
     delete pcoinsTip;
+    pcoinsTip = nullptr;
```

Comment:
https://github.com/dashpay/dash/pull/2763/commits/4ef306d58e110920d6ce8bf5316a811638d1a304

Reply:
Maybe it's time to backport ff6a834fc32b25376e221fefa8c2a2565b4167cc already now instead of waiting for bitcoin#10179. This repeating of initialization and destruction has caused too much head aches already. @PastaPastaPasta can you add this single commit to this PR?